### DISCUSSION TITLE
Unsupported RPC request eth_submitHashrate

### DISCUSSION THREAD

**Discussion by XIII39** [coinfoundry/miningcore#1052] [Q&A]
I keep getting this error in the console, I'm assuming it's not all that important but I can't seem to find the root of the problem.

Unsupported RPC request: {"jsonrpc":"2.0","method":"eth_submitHashrate","params Unsupported RPC request: {"jsonrpc":"2.0","method":"eth_submitHashrate","params":{"0x0.....

My geth http.api=eth,net,web3,miner & ws.api=eth,net,web3,miner

Am I missing something to make the jsonrpc eth_submitHashrate work?

**Comment by oliverw** [comment]
This message is completely benign and nothing to worry about.

**Reply by XIII39** [reply]
Yeah, as I understand it eth_submithashrate pretty much deprecated. Is there a way to suppress the error so I don't have to see it everytime I look at the console?

**Comment by oliverw** [comment]
Will be suppressed in next release.
